GitUI/CommandsDialogs/FormMailMap.cs

```diff
@@ -55,7 +58,7 @@ private void LoadFile()
                 var path = _fullPathResolver.Resolve(".mailmap");
                 if (File.Exists(path))
                 {
-                    _NO_TRANSLATE_MailMapText.ViewFileAsync(path);
+                    _NO_TRANSLATE_MailMapText.ViewFileAsync(path!);
```

Comment:
`File.Exists` is not annotated in .NET Framework. However if it returns `true`, then `path` is definitely not null. Suppress the warning here. When moving to .NET 5+, this will not be necessary.